infra/gcp/clusters/projects/k8s-infra-prow-build/prow-build/README.md

```diff
@@ -4,6 +4,25 @@ These terraform resources define a GCP project containing a GKE cluster
 intended to serve as a "build cluster" for prow.k8s.io. There are also
 some service accounts defined for use by pods within the cluster.
 
+## Accessing the control plane
+
+The k8s-infra-prow-build cluster is a [private GKE cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters).
+It cannot be directly accessed with kubectl from outside the GCP project,
+cloud-shell must first be used as follows:
+```shell
+# Start a cloud shell
+gcloud alpha cloud-shell ssh --project=k8s-infra-prow-build
+
+# Login to set the authenticated user for gcloud in the cloud shell session.
```

Comment:
As of https://github.com/kubernetes/k8s.io/pull/1727 this part is no longer true, so here's my suggestion to make this more relevant.  Can move the bookmarks to the bottom of the page.
```suggestion
Access to the k8s-infra-prow-build project hosting the cluster is granted by membership in one of two @kubernetes.io groups:
- [k8s-infra-prow-oncall@kubernetes.io][k8s-infra-prow-oncall@]: grants [`roles/owner`][roles/owner] access
- [k8s-infra-prow-viewers@kubernetes.io][k8s-infra-prow-viewers@]: grants [`prow.viewer`][roles/prow.viewer] access

If you are not a member of either of these groups, please [follow these instructions to join][join-groups]

[k8s-infra-prow-oncall]: https://github.com/kubernetes/k8s.io/blob/3a1aea1652f02a95253402bde2bca63cb4292f8e/groups/groups.yaml#L647-L670
[k8s-infra-prow-viewers]: https://github.com/kubernetes/k8s.io/blob/3a1aea1652f02a95253402bde2bca63cb4292f8e/groups/groups.yaml#L672-L699
[roles/owner]: https://cloud.google.com/iam/docs/understanding-roles#basic-definitions
[roles/prow.viewer]: https://github.com/kubernetes/k8s.io/blob/main/infra/gcp/roles/prow.viewer.yaml
[join-groups]: https://github.com/kubernetes/k8s.io/tree/main/groups#making-changes

```shell
# Login to set the authenticated user for gcloud
```

Reply:
Awesome, added these changes.